src/app.js

```diff
@@ -3,18 +3,72 @@ const morgan = require('morgan')
 
 const router = require('./routes')
 
+const db = require('./db')
+
 const app = express()
 
 // Middleware
 app.use(express.json())
-if(process.env.NODE_ENV === 'development'){
+if (process.env.NODE_ENV === 'development') {
     app.use(morgan('dev'))
 }
 app.get('/', (req, res) => {
     res.send('Hello friend')
 })
 
+// CREATE USER ENDPOINT
+
+app.post('/users', async (req, res) => {
```

Comment:
@beblicarl please is this within the scope or a personal addition? I was thinking this was being worked on in another story here https://github.com/tbf-challenge/teamwork-be/pull/65/files

Reply:
Yes, that's true